test/unit-tests/broker-connection.test.js

```diff
@@ -0,0 +1,26 @@
+const expect = require('chai').expect;
+const io = require('socket.io-client');
+
+describe('Broker connection handling', () => {
+    let client, dashboardServer;
+
+    before((done) => {
+        dashboardServer = require('../../dashboard/server/index');
```

Comment:
Any specific reason to import in `before` block?

Reply:
This was required before when we were clearing the ```require.cache``` but there is no need to use it now, it has been removed!